DisplaySettings/DisplaySettings.cpp

```diff
@@ -1342,7 +1342,53 @@ namespace WPEFramework {
             try
             {
                 device::VideoOutputPort &vPort = device::Host::getInstance().getVideoOutputPort(videoDisplay);
-                response["resolution"] = vPort.getResolution().getName();
+		int width = 0;
+		int height = 0;
+		string res = vPort.getResolution().getName();
+		if(strcmp(res.c_str(),"480i")== 0 || strcmp(res.c_str(),"480p")== 0)
+		{
+		    width =  720;
+		    height = 480;
+		}
+		else if(strcmp(res.c_str(),"576p50")== 0)
+		{
+		    width =  720;
+		    height = 576;
+		}
+		else if(strcmp(res.c_str(),"720p")== 0 || strcmp(res.c_str(),"720p50")== 0)
+		{
+		    width =  1280;
+		    height = 720;
+		}
+		else if(strcmp(res.c_str(),"768p")== 0)
+		{
+		    width =  1366;
+		    height = 768;
+		}
+		else if(strcmp(res.c_str(),"1080p24")== 0 || strcmp(res.c_str(),"1080p")== 0 || strcmp(res.c_str(),"1080i50")== 0 || strcmp(res.c_str(),"1080i")== 0)
+		{
+		    width =  1920;
+		    height = 1080;
+		}
+		else if(strcmp(res.c_str(),"2160p30")== 0 || strcmp(res.c_str(),"2160p60")== 0)
+		{
+		    width =  3840;
+		    height = 2160;
+		}
+		else if(strcmp(res.c_str(),"4096x2160p24")== 0 || strcmp(res.c_str(),"4096x2160p25")== 0 || strcmp(res.c_str(),"4096x2160p30")== 0 || strcmp(res.c_str(),"4096x2160p50")== 0 || strcmp(res.c_str(),"4096x2160p60")== 0)
```

Comment:
could we simplyfy this and just compare start with 4096x2160 so we dont need to add all resolution ? Same applies to aall other switch cases as well

Reply:
used rfind Simplyfy the compare start with all resolutions.